dune/stuff/common/algorithm.hh

```diff
@@ -8,21 +8,23 @@
 
 #include <utility>
 
-namespace Dune {
-namespace Stuff {
-namespace Common {
+namespace Dune
+{
+namespace Stuff
+{
+namespace Common
+{
 
 //! same as std::copy_if, but with move
-template<class InputIt, class OutputIt, class UnaryPredicate>
-OutputIt move_if(InputIt first, InputIt last,
-                 OutputIt d_first, UnaryPredicate pred)
+template < class InputIt, class OutputIt, class UnaryPredicate >
```

Comment:
Not a fan of spaces inside template myself, but can live with either way. Really don't like the inconsistency you suggest in having a space inside, but not before.